packages/rrweb/src/replay/index.ts

```diff
@@ -293,23 +299,27 @@ export class Replayer {
         this.adoptedStyleSheets = [];
       }
 
-      if (this.mousePos) {
-        this.moveAndHover(
-          this.mousePos.x,
-          this.mousePos.y,
-          this.mousePos.id,
-          true,
-          this.mousePos.debugData,
-        );
-        this.mousePos = null;
-      }
-
-      if (this.touchActive === true) {
-        this.mouse.classList.add('touch-active');
-      } else if (this.touchActive === false) {
-        this.mouse.classList.remove('touch-active');
+      for (const [touchId, value] of Object.entries(this.multiTouchMap)) {
+        const { mouse } = value;
```

Comment:
Does destructuring inline work?

```suggestion
      for (const [touchId, {mouse}] of Object.entries(this.multiTouchMap)) {
```